Source/SwiftLintFramework/Rules/RuleConfigs/NameConfig.swift

```diff
@@ -14,11 +14,19 @@ public struct NameConfig: RuleConfig, Equatable {
     var excluded: Set<String>
 
     var minLengthThreshold: Int {
-        return max(minLength.warning, minLength.error)
+        if let minLengthError = minLength.error {
+            return max(minLength.warning, minLengthError)
+        } else {
+            return minLength.warning
+        }
     }
 
     var maxLengthThreshold: Int {
-        return min(maxLength.warning, maxLength.error)
```

Comment:
This can just be `return min(maxLength.warning, maxLength.error ?? maxLength.warning)`